cmd/internal/dgather.go

```diff
@@ -282,7 +285,7 @@ func GenerateJobConsumersData(currentJob *manifest.Job, jobReleaseSpecs map[stri
 }
 
 // RenderJobBPM per job and add its value to the jobInstances.BPM field
-func RenderJobBPM(currentJob manifest.Job, jobInstances []manifest.JobInstance, baseDir string, manifestName string) error {
+func RenderJobBPM(currentJob *manifest.Job, jobInstances []manifest.JobInstance, baseDir string, manifestName string, log *zap.SugaredLogger) error {
```

Comment:
Don't pass a `log`, use a `Context`
See here:  https://github.com/cloudfoundry-incubator/cf-operator/blob/master/pkg/kube/util/ctxlog/context.go

Every function should have a context, and the `log` object is retrieved using `ExtractLogger`

Reply:
done